Services/CO.CDP.Organisation.WebApi/Api/Organisation.cs

```diff
@@ -22,6 +22,24 @@ public static class EndpointExtensions
 {
     public static void UseOrganisationEndpoints(this WebApplication app)
     {
+        app.MapGet("/admin/organisations",
+                async ([FromQuery] string type, [FromQuery] int limit, [FromQuery] int skip, IUseCase<(string, int, int), IEnumerable<Model.ApprovableOrganisation>> useCase) =>
+                await useCase.Execute((type, limit, skip))
```

Comment:
Given that there's a scope for more query parameters added in future, I think we'd be better off with a dedicated query record here, instead of a tupple, i.e. `OrganisationsQuery`. This way it will be easier to introduce additional query params later.

Reply:
Ok. I'll get this changed.